sandy/sections/mf2.py

```diff
@@ -45,119 +46,118 @@ def read_mf2(tape, mat):
-    {'AWRI': 57.43561, 'L': 3, 'NJS': 2, 
-     'RES_PAR': [{'D': 8033.33, 'AJ': 2.5, 'AMUN': 1.0, 'GN0': 0.482, 'GG': 0.7, '0': 0.0}, 
-                 {'D': 6025.0, 'AJ': 3.5, 'AMUN': 1.0, 'GN0': 0.3615, 'GG': 0.7, '0': 0.0}]}
-    
-    
+    >>> print (dic["NIS"][26058]["NER"][(350000.0, 3000000.0)]["L"][3])
+    {'AWRI': 57.43561, 'L': 3, 'NJS': 2,
+     'RES_PAR': [{'D': 8033.33, 'AJ': 2.5, 'AMUN': 1.0, 'GN0': 0.482, 'GG': 0.7},
+                 {'D': 6025.0, 'AJ': 3.5, 'AMUN': 1.0, 'GN0': 0.3615, 'GG': 0.7}]}
+
+
     Resonance parameters of the iron 54
     LRU = 1 LRF = 7
     >>> tape = sandy.get_endf6_file("endfb_80", "xs", 260540)
     >>> dic = sandy.read_mf2(tape, 2625)
-    >>> print (dic["NIS"]["ISO"][26054]["NER"][(1e-05, 1036000.0)]["J"][2]["SPIN_GROUP"])
-    [{'PPI': 1.0, 'L': 0.0, 'SCH': 0.0, 'BND': 0.0, 'APE': 0.0, 'APT': 0.0}, 
+    >>> print (dic["NIS"][26054]["NER"][(1e-05, 1036000.0)]["J"][2]["SPIN_GROUP"])
+    [{'PPI': 1.0, 'L': 0.0, 'SCH': 0.0, 'BND': 0.0, 'APE': 0.0, 'APT': 0.0},
      {'PPI': 2.0, 'L': 1.0, 'SCH': 0.5, 'BND': 0.0, 'APE': 0.54373, 'APT': 0.54373}]
-    
-    
+
+
     Resonance parameters of the Uranium 235
     LRU = 2 LRF = 2
     >>> tape = sandy.get_endf6_file("jeff_33", "xs", 922350)
     >>> dic = sandy.read_mf2(tape, 9228)
-    >>> print (dic["NIS"]["ISO"][92235]["NER"][(2250.0, 46200.0)]['L'][0]["J"][0]["RES_PAR"][0])
+    >>> print (dic["NIS"][92235]["NER"][(2250.0, 46200.0)]['L'][0]["J"][0]["RES_PAR"][0])
     {'ES': 2250.0, 'D': 1.058, 'GX': 0.0, 'GN0': 0.000107789, 'GG': 0.038513, 'GF': 0.40102}
-    
+
     """
     df = tape._get_section_df(mat, mf, mt)
     out = {
-            "MAT": mat,         
-            "MF": mf,
-            "MT": mt,
-            }
+        "MAT": mat,
+        "MF": mf,
+        "MT": mt,
+    }
     i = 0
     C, i = sandy.read_cont(df, i)
     add = {
-            "ZA": C.C1,             #designation for an isotope
-            "AWR": C.C2,            #AWR is defines as the ratio of the mass of the material to that of the neutron
-            }
+        "ZA": C.C1,  # designation for an isotope
+        "AWR": C.C2,  # AWR is defines as the ratio of the mass of the material to that of the neutron
+    }
     NIS = int(C.N1)
-    
+
     out.update(add)
     P = {}
-    for l in range(NIS): 
+    for l in range(NIS):
         M = {}
         NER1 = {}
         ISO = {}
         C, i = sandy.read_cont(df, i)
         header1 = {
-                "ZAI": C.C1,            #designation for an isotope
-                "ABN": C.C2,            #Abundance of an isotope in the material 
-                "LFW": C.L2,            #indication whether average fission wifths are given in the unresolbed resonance region
-                } 
+            "ABN": C.C2,  # Abundance of an isotope in the material
+            "LFW": C.L2,  # indication whether average fission wifths are given in the unresolbed resonance region
```

Comment:
Item L2 of a cont record like LFW should already be a an integer, why do you do `LFW = int(C.L2)` below?